deps/build.jl

```diff
@@ -66,7 +64,8 @@ provides(BuildProcess, (@build_steps begin
         FileRule(joinpath(libdir(libfftw), "libfftw." * Libdl.dlext), @build_steps begin
             CreateDirectory(libdir(libfftw))
             `$(joinpath(".", "configure")) $general_config $fftw_config $fftw_enable_single`
-            `$MAKE`
+            `$MAKE_CMD`
+            `$MAKE_CMD install`
```

Comment:
Ah, let me illuminate a bit: FFTW builds two versions of its libraries, one for `double` precision (`libfftw.so`), one for `single` precision (`libfftwf.so`). and so you have to run through the `configure`, `make`, `make install` separately for each.  So you're gonna have to create two `FileRule`'s here if you want the single and double versions to both exist (which you do, if you want to mimic what is happening in Base Julia).

Also you're defining a `FileRule` for `libfftw.so` here, but then building `libfftwf.so`, so you should probably fix that.